west.yml

```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 8d93a161567dcc525771f70692587e6a37947b71
+      revision: 226b676f94f6d0999b3199a6723057f533754ac7
```

Comment:
Is this supposed to be on its own commit?
Or why is it sneaked in?
Commit message does not describe the change.

Reply:
well, manifest change pulls same kind of changes